workloads/api-load/env.sh

```diff
@@ -1,7 +1,6 @@
 # Common
 
 TEST_CLEANUP=${TEST_CLEANUP:-true}
-export ES_SERVER=${ES_SERVER:-https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443}
 export ES_INDEX=ripsaw-api-load
```

Comment:
Let's have a placeholder with the default value `export ES_SERVER=${ES_SERVER:-""}`

Reply:
The code flow and behavior would be the same. Any specific reason to add an empty string?